features/routing/operator.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Testing Ingress Operator related scenarios
+
+  # @author hongli@redhat.com
+  # @case_id OCP-27594
+  @admin
+  Scenario: set namespaceOwnership of routeAdmission to InterNamespaceAllowed
+    Given the master version >= "4.4"
+    And I have a project
+    And I store default router subdomain in the :subdomain clipboard
+    Given I switch to cluster admin pseudo user
+    And admin ensures "test-27594" ingresscontroller is deleted from the "openshift-ingress-operator" project after scenario
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/routing/operator/ingressctl-nsowner.yaml" replacing paths:
+      | ["metadata"]["name"] | test-27594                                    |
+      | ["spec"]["domain"]   | <%= cb.subdomain.gsub("apps","test-27594") %> |
+    Then the step should succeed
+
+    # check the env in the router pod
+    Given I use the "openshift-ingress" project
+    And a pod becomes ready with labels:
+      | ingresscontroller.operator.openshift.io/deployment-ingresscontroller=test-27594 |
+    And evaluation of `pod.name` is stored in the :router_pod clipboard
+    Given I wait up to 30 seconds for the steps to pass:
+    """
+    When I execute on the pod:
+      | env |
+    Then the output should contain:
+      | ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true |
+    """
+
+    # create route in the first namespace
+    Given I switch to the first user
+    And I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/routing/caddy-docker.json |
```

Comment:
@lihongan v3-testfiles has been retired...pls use `<%= BushSlicer::HOME %>/testdata/...` for all of your call to `https://raw.githubusercontent.com/openshift-qe/v3-testfiles`

Reply:
Good catch! Thank you @pruan-rht .
Updated and here is the log:
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/93842/console